frontend/website/src/routes/+layout.svelte

```diff
@@ -19,7 +19,9 @@
 	.app {
```

Comment:
Don't mix Tailwind and regular CSS.

Tailwind is optimized at the build step and trimmed down while regular CSS isn't. If you wish to use CSS alongside Tailwind please use a preprocesses like [Sass](https://sass-lang.com/).